pkg/operator/ceph/controller/spec.go

```diff
@@ -601,6 +603,40 @@ fi`,
 	}
 }
 
+// GenerateReadinessProbeExecDaemon generates a readiness probe that makes sure a daemon is ready
+// to process traffic, and that it returns 0
+func GenerateReadinessProbeExecDaemon(daemonType, daemonID string) *v1.Probe {
+	confDaemon := getDaemonConfig(daemonType, daemonID)
+	command, msg := confDaemon.buildDaemonReadinessCheckCommand()
+	return &v1.Probe{
+		ProbeHandler: v1.ProbeHandler{
+			Exec: &v1.ExecAction{
+				// Run with env -i to clean env variables in the exec context
+				// This avoids conflict with the CEPH_ARGS env
+				//
+				// Example:
+				// env -i sh -c "ceph --admin-daemon /run/ceph/ceph-mgr.a.asok mgr_status"
+				//
+				Command: []string{
+					"env",
+					"-i",
+					"sh",
+					"-c",
+					fmt.Sprintf(`outp="$(ceph --admin-daemon %s %s 2>&1)"
```

Comment:
Is `$outp` used somewhere, or why assign the output to it? If it is going to be used somewhere else, perhaps use a complete variable name

Reply:
This variable is not used. I'll just remove it.

Reply:
Fixed.